Composer/packages/client/src/pages/design/exportSkillModal/index.tsx

```diff
@@ -147,7 +218,16 @@ const ExportSkillModal: React.FC<ExportSkillModalProps> = ({ onSubmit, onDismiss
             <div>
               {buttons.map(({ disabled, primary, text, onClick }, index) => {
                 const Button = primary ? PrimaryButton : DefaultButton;
-                const isDisabled = typeof disabled === 'function' ? disabled({ manifest: skillManifest }) : !!disabled;
+
+                let isDisabled = false;
+                if (typeof disabled === 'function') {
+                  isDisabled =
+                    editorStep === ManifestEditorSteps.SELECT_MANIFEST
+                      ? disabled({ manifest: skillManifest })
+                      : disabled({ publishTargets });
+                } else {
+                  isDisabled = !!disabled;
```

Comment:
Why is this else required?

Reply:
disabled type can be a function or not, so we should set `isDisabled` depend on `disabled`